examples/resources/hcp_consul_cluster/federation.tf

```diff
@@ -1,19 +1,27 @@
-resource "hcp_hvn" "example" {
+resource "hcp_hvn" "east" {
   hvn_id         = "hvn"
   cloud_provider = "aws"
-  region         = "us-west-2"
+  region         = "us-east-1"
   cidr_block     = "172.25.16.0/20"
 }
 
+resource "hcp_hvn" "west" {
+  hvn_id         = "hvn"
+  cloud_provider = "aws"
+  region         = "us-west-2"
+  cidr_block     = "172.26.16.0/20"
+}
+
 resource "hcp_consul_cluster" "primary" {
-  hvn_id     = hcp_hvn.example.hvn_id
+  hvn_id     = hcp_hvn.east.hvn_id
   cluster_id = "consul-cluster-primary"
   tier       = "development"
 }
 
 resource "hcp_consul_cluster" "secondary" {
-  hvn_id       = hcp_hvn.example.hvn_id
-  cluster_id   = "consul-cluster-secondary"
-  tier         = "development"
-  primary_link = hcp_consul_cluster.primary.self_link
+  hvn_id                  = hcp_hvn.west.hvn_id
+  cluster_id              = "consul-cluster-secondary"
+  tier                    = "development"
+  primary_link            = hcp_consul_cluster.primary.self_link
+  auto_hvn_to_hvn_peering = true
```

Comment:
Does `auto_hvn_to_hvn_peering` have to be set to true on all Consul clusters in a federation? Or just the secondary? Worth calling out in the guide.

Reply:
Good point! Will add a note in the guide and maybe in the docs as well.